unifi/Dockerfile

```diff
@@ -1,21 +1,37 @@
-ARG BUILD_FROM=ghcr.io/hassio-addons/ubuntu-base:8.2.0
+ARG BUILD_FROM=ghcr.io/hassio-addons/ubuntu-base:9.0.4
 # hadolint ignore=DL3006
 FROM ${BUILD_FROM}
 
+ARG BUILD_ARCH=amd64
+
 # Set shell
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Setup base system
+
+# mongodb-org-server is installed from the community servers as it's gone from
+# repositories for newer ubuntu versions.
+#
+# Beware of the licence change, previously the addon used mongodb version 3.6.9
+# released under the AGPL, while 4.0.3 and newer are released under the non-free SSPL
+# https://en.wikipedia.org/wiki/MongoDB#Licensing
```

Comment:
```suggestion
```

Reply:
Not sure what I messed up, I was sure I had removed this yesterday. Should be gone for good now, thanks for your commit!